docs/Early Access - Using Global Profile Configuration.md

```diff
@@ -1,36 +1,131 @@
 
-Read through the [Zowe Explorer ReadMe file](https://github.com/zowe/vscode-extension-for-zowe/blob/master/packages/zowe-explorer/README.md) to familiarize yourself with the capabilities of Zowe Explorer, and you are ready to use Zowe Explorer.
+## Install the @Next Release version
 
-### Install the vNext Release version
+Install @Next.
 
-1. Get the latest pre-release version from the Zowe Explorer [Github Releases page](https://github.com/zowe/vscode-extension-for-zowe/releases)
-2. Within VS Code, use **File > Preferences > Extensions > Install from vsix**
-3. Select the files you downloaded
-4. Reload your VS Code window.
+**Follow these steps**:
+
+1. Download the latest [@Next release version](https://github.com/zowe/vscode-extension-for-zowe/releases) from the Zowe Explorer Github release page.
+2. Open VS Code.
+3. Navigate to **File** > **Preferences** > **Extensions** > **Install from vsix\*\***.
+4. Select the .vsix file to install.
+5. Reload your VS Code window.
 
 ### Load a Profile
 
+**Follow these steps**:
+
 1. Navigate to the explorer tree.
 2. Hover over **DATA SETS**, **USS**, or **JOBS**.
 3. Click the **+** icon.
 4. From the drop-down menu, select the profile that you want to use.
 
-You can now use all the functionalities of the extension.
+You can now use your global or project-specific profile.
+
+## Profile Configuration
+
+The global profile functionality simplifies profile management by enabling you to edit, store, and share mainframe configuration details in one location. You can use a text editor or an IDE to populate configuration files with connection details for your mainframe services. By default, your global configuration profile is located in .zowe home folder, whereas a project-level configuration file is located in the main directory of your project.
+
+**Note**: A project context takes precedence over global configuration.
+
+### Manage a Profile
+
+You can edit your project-level or global configuration profiles.
+
+**Follow these steps**:
+
+1. Right-click on your profile.
+2. Select the **Add**, **Update**, or **Delete Profile** options to edit the global `zowe.config.json` file.
+
+   **Tip**: Use the Intellisense prompts if you need assistance with filling parameters in the .json file.
+
+3. Reload your VS Code window by clicking **View** > **Command Palette** > **Developer: Reload Window** so that the changes take effect.
+
+You successfully edited your configuration profile.
+
+### Sample Profile Configuration
+
+View the profile configuration sample. In the sample, the default `lpar1.zosmf` profile will be loaded upon activation.
 
-### Add, Update, and Delete a Profile
+You can use the sample to customize you profile configuration file. Ensure that you edit the `host` and `port` values before you work in your environment.
```

Comment:
small typo you -> your profile configuration file

Reply:
oh... Nice catch! let me fix that